modules/apps/google-places/google-places-web/src/main/resources/content/Language.properties

```diff
@@ -1,4 +1,3 @@
 ensure-that-both-googles-places-api-and-maps-javascript-api-are-enabled-for-that-key=Ensure that both Google's Places API and Maps JavaScript API are enabled for that key.
-google-places=Google Places
-google-places-api-key=Google Places API Key
+google-places-api=Google Places API
```

Comment:
Was this change proposed by Brian or maybe the design team? My concern here is that Brian, back when this was created, had specific demands about the naming of this setting.

Reply:
Not really. It came from my mind due to the recent changes. :grin: 

Previously, we didn't have a dedicated category to Google Places settings, actually, it was inside the category Thirty Party, as you can see below:

![Previous Google Places](https://user-images.githubusercontent.com/6729287/131575544-dc47ce4c-83bf-49c0-bb25-8680860e7d06.png)

When I created the new category then I realized that the phrase **Google Places API** will be repeated 3 times, because it'll be the category name, the setting title name and the setting name. Therefore, in order to prevent this repeating then I decided to propose something different, as you can see below: :smile: 

![New Google Places](https://user-images.githubusercontent.com/6729287/131576067-81f5bca4-85f1-4bb4-903e-1f88c57f3646.png)

Reply:
Got it. Thanks for the screenshots. They helped a lot.

Personally, I don't think that's something to worry about, @natocesarrego. You see, we have examples of settings being repetitive:

<img width="1344" alt="Screen Shot 2021-08-31 at 18 50 40" src="https://user-images.githubusercontent.com/11450348/131580489-1ba282f3-f8ad-4631-b359-56713ee0103e.png">

Also I'm not sure if "Google Places API" is helpful to the user since both Google Places API and Maps JavaScript API must be enabled for the same key. I do like your proposal on changing the input's title though. Perhaps, it would make more sense if we had something like Google Places (the new category) > Credentials (setting) > Key (or API Key for the input's title). What do you think?

Reply:
I think I got your point, @rodrigopaulino. :slightly_smiling_face:  Maybe credentials could be a good idea, I'll process it and resend.

Thanks.